code/datums/components/butchering.dm

```diff
@@ -27,13 +27,29 @@
 		if(butchering_enabled && (can_be_blunt || source.get_sharpness()))
 			INVOKE_ASYNC(src, .proc/startButcher, source, M, user)
 			return COMPONENT_ITEM_NO_ATTACK
+	if(user.a_intent == INTENT_HARM && ishuman(M) && source.get_sharpness())
+		var/mob/living/carbon/human/H = M
+		if((H.health <= H.crit_threshold || (user.pulling == H && user.grab_state >= GRAB_NECK) || H.IsSleeping()) && user.zone_selected == BODY_ZONE_HEAD)
+			INVOKE_ASYNC(src, .proc/startNeckSlice, source, H, user)
+			return COMPONENT_ITEM_NO_ATTACK
 
 /datum/component/butchering/proc/startButcher(obj/item/source, mob/living/M, mob/living/user)
 	to_chat(user, "<span class='notice'>You begin to butcher [M]...</span>")
 	playsound(M.loc, butcher_sound, 50, TRUE, -1)
 	if(do_mob(user, M, speed) && M.Adjacent(source))
 		Butcher(user, M)
 
+/datum/component/butchering/proc/startNeckSlice(obj/item/source, mob/living/carbon/human/H, mob/living/user)
+	H.visible_message("<span class='danger'>[user] is slitting [H]'s throat!</span>", \
+					"<span class='userdanger'>Your throat is being slit by [user]!</span>", \
+					"<span class='userdanger'>Something is cutting into your neck!</span>")
```

Comment:
blind_message is shown to all mobs nearby so this doesn't really make sense.
```
	user.visible_message("<span class='danger'>[user] is slitting [H]'s throat!</span>", \
					"<span class='danger'>You start slicing [H]'s throat!</span>", \
					"<span class='hear'>You hear a cutting noise!</span>", ignored_mobs = H)
	H.show_message("<span class='userdanger'>Your throat is being slit by [user]!</span>", MSG_VISUAL, "<span class = 'userdanger'>Something is cutting into your neck!</span>", NONE)
```
This properly covers ALL cases (which tbh is better than most messages)

Reply:
Oops, I miss read the visible_message code. Thank you for this.